src/edu/csus/ecs/pc2/core/imports/clics/CLICSAward.java

```diff
@@ -52,11 +50,11 @@ public CLICSAward() {
      * @param citation ex. Contest winner
      * @param team_ids lsit of ids
      */
-    public CLICSAward(String id, String citation, List<String> team_ids) {
+    public CLICSAward(String id, String citation, String [] team_ids) {
         super();
         this.id = id;
         this.citation = citation;
-        this.team_ids = (String[]) team_ids.toArray(new String[team_ids.size()]);
+        this.team_ids = team_ids;
```

Comment:
This change should be reverted as part of fixing the issue referred to in my previous comments.